arch/arm/src/samv7/sam_qspi.c

```diff
@@ -1089,7 +1089,7 @@ static uint32_t qspi_setfrequency(struct qspi_dev_s *dev, uint32_t frequency)
 #endif
   uint32_t regval;
 
-  spiinfo("frequency=%d\n", frequency);
+  spiinfo("frequency=%"PRId32"\n", frequency);
```

Comment:
`PRIu32` would be a better candidate for unsigned types.

Reply:
You are correct. I replaced it to be the same as so far, only to fix the warning. Let me modify that. It should be short patch again.